lib/timeline/component/Group.js

```diff
@@ -360,8 +354,15 @@ Group.prototype.redraw = function(range, margin, forceRestack, returnQueue) {
 
     // update vertical position of items after they are re-stacked and the height of the group is calculated
     (function() {
-      return resized = this._updateItemsVerticalPosition.bind(this)(resized, margin)
-    }).bind(this)
+      this._updateItemsVerticalPosition.bind(this)(resized, margin)
+    }).bind(this),
+
+    function() {
+      if (!this.isVisible && this.height) {
+        resized = false;
```

Comment:
But `resized` is a boolean, right? I still don't understand why previous went wrong, but it really isn't important.